dace/transformation/pattern_matching.py

```diff
@@ -64,6 +64,23 @@ def type_match(node_a, node_b):
     return isinstance(node_a['node'], type(node_b['node']))
 
 
+def type_or_class_match(node_a, node_b):
+    """ Checks whether the node types of the inputs match.
```

Comment:
What exactly does it mean that "types of the inputs match"? Maybe this needs a bit of context?

Reply:
Thanks, fixed.